fe/fe-core/src/main/java/com/starrocks/sql/analyzer/QueryAnalyzer.java

```diff
@@ -597,6 +597,11 @@ private Expr analyzeJoinUsing(List<String> usingColNames, Scope left, Scope righ
     private Scope analyzeFrom(SelectStmt node, AnalyzeState analyzeState, Scope scope) {
         Relation sourceRelation = null;
         TableRef lastTableRef = null;
+
+        if (node.getTableRefs().size() == 1 && node.getTableRefs().get(0) instanceof FunctionTableRef) {
```

Comment:
What if node.getTableRefs().size() is 2, and one fo table ref is `FunctionTableRef `?

Reply:
Analyze success